src/components/MessageInput/MessageInputTypes.ts

```diff
@@ -0,0 +1,52 @@
+type SetEmojiPickerIsOpenAction = {
```

Comment:
I definitely think that it is way better to extract the types to their own files like this. 👍 I only wonder if we should go with one per component or if we could have one type file per "namespace": Like `src/components/MessageInput/types.ts`. But that's something we can discuss and adjust along the way.

Reply:
Ah yes that also makes sense, I wasn't really sure what to call this so if we just go with types.ts, I won't have to care about naming anymore :)